src/fides/cli/utils.py

```diff
@@ -144,70 +143,47 @@ def register_user(config: FidesConfig, email: str, organization: str) -> None:
     )
 
 
-def check_and_update_analytics_config(
-    config: FidesConfig, config_path: str
-) -> FidesConfig:
+def request_analytics_consent(config: FidesConfig, opt_in: bool = False) -> FidesConfig:
     """
-    Verify that the analytics opt-out value is populated. If not,
-    prompt the user to opt-in to analytics and update the config
-    file with their preferences if needed.
-
-    NOTE: This doesn't handle the case where we've collected consent for this
-    CLI instance, but are connected to a server for the first time that is
-    unregistered. This *should* be something we can detect and then
-    "re-prompt" the user for their email/org information, but right
-    now a lot of our test automation runs headless and this kind of
-    prompt can't be skipped otherwise.
+    Request the user's consent for analytics collection.
+
+    This function should only be called when specifically wanting to ask
+    for the user's consent, otherwise it will ask repeatedly for consent
+    unless they've opted in.
     """
 
-    config_updates: Dict[str, Dict] = {}
-    if config.user.analytics_opt_out is None:
-        click.echo(OPT_OUT_COPY)
-        config.user.analytics_opt_out = bool(
-            input(OPT_OUT_PROMPT + "\n").lower() == "n"
+    # Check the analytics opt out env var
+    # if getenv("FIDES__USER__ANALYTICS_OPT_OUT", "true") != "false":
+    #     return config
```

Comment:
Is this commented code supposed to be here?

```suggestion

```

Reply:
yes I've fixed that....forgot to re-enable. Newer commits will respect the env var :) 

disabled it for testing since its automatically set in the docker container